backend/app/assets/javascripts/spree/backend/sticky.coffee

```diff
@@ -0,0 +1,106 @@
+$.fn.offsetBottom = ->
+  $(this).offset().top + $(this).height()
+
+class Viewport
```

Comment:
I don't know enough about the javascripts to suggest alternatives here, but I feel like this should exist, and if something doesnt that gives us the same behavior with the same quality, or a minimal amount of LOC I'd probably like to see this extracted into a common plugin, as it sounds like it'd be useful for others. Maybe out to solidus-contrib? What do people think about this?